railties/lib/rails/api/task.rb

```diff
@@ -9,7 +10,10 @@ class Task < RDoc::Task
             README.rdoc
             lib/active_support/**/*.rb
           ),
-          exclude: "lib/active_support/vendor/*"
+          exclude: %w(
+            lib/active_support/i18n_railtie.rb
+            lib/active_support/xml_mini/*
+          )
```

Comment:
Yeah, you're right !

I've added some `:enddoc:` directives ; there are still `Encoding` and `Mysql2` which are present in the sidebar but since `:enddoc:` stops documentation for the whole file and since these constants are referenced at the very top of files, we can't remove them for now as RDoc ignores `:stopdoc:` directive at the top level.